scripts/post.lua

```diff
@@ -0,0 +1,22 @@
+local pipelineDepth = 15
+
+function init(args)
+   wrk.method = "POST"
+   wrk.body   = "foo=bar&baz=quux"
+   wrk.headers["Content-Type"] = "application/x-www-form-urlencoded"
+
+   if args[1] ~= nil then
+      pipelineDepth = args[1]
+   end
+
```

Comment:
The script should print the pipelineDepth like `pipeline.lua`, so its included in the output.